kylin-spark-project/kylin-spark-engine/src/main/scala/org/apache/kylin/engine/spark/builder/CubeDictionaryBuilder.scala

```diff
@@ -70,7 +70,7 @@ class DFDictionaryBuilder(val dataset: Dataset[Row],
     val columnName = ref.identity
     logInfo(s"Start building global dictionaries V2 for column $columnName.")
```

Comment:
V2 need to be removed?

Reply:
Right, there're still some logs need to be modified.